src/test/java/net/openhft/chronicle/queue/impl/single/stress/RollCycleMultiThreadTest.java

```diff
@@ -329,11 +330,13 @@ public synchronized Integer call() {
                 if (!dc.isPresent())
                     return -2;
 
-                StringBuilder sb = Wires.acquireStringBuilder();
-                dc.wire().read("say").text(sb);
+                try (ScopedResource<StringBuilder> stlSb = Wires.acquireStringBuilderScoped()) {
```

Comment:
appreciate this is maybe going beyond the scope of this change, but maybe just have a StringBuilder field (from memory this class is per-thread?)